node:
  lidarscan_topic: "/velodyne/points"
  map_publish_rate: 10.0             # [Hz]
  pose_update_rate: 10.0             # [Hz]
  remove_backpoints: true            # [bool], Remove remoter points
  debug_mode: false

frame_id:
  robot_frame: "base_link"
  map_frame: &map_frame "map" 

height_mapper:
  estimator_type: "StatMean"   # ["StatMean", "KalmanFilter", "MovingAverage"]
  frame_id: *map_frame
  map_length_x: 10.0                 # [meter]
  map_length_y: 10.0                 # [meter]
  grid_resolution: 0.1               # [meter]
  min_height_threshold: -0.2         # [meter]
  max_height_threshold: 1.5          # [meter]

traversability_network:
  model_path: "$(env HOME)/Downloads/LeSTA_campus_road_libtorch.pt"
  input_dimension: 4
  feature_fields:
    - step
    - slope
    - roughness
    - curvature
    # - elevation_variance
  binary_threshold: 0.5
